bindings/rabbitmq/rabbitmq.go

```diff
@@ -464,8 +465,20 @@ func (r *RabbitMQ) handleMessage(ctx context.Context, handler bindings.Handler,
 				r.logger.Info("Input binding channel closed")
 				return
 			}
+
+			metadata := make(map[string]string)
```

Comment:
```suggestion
			metadata := make(map[string]string, len(d.Headers))
```

Small memory optimization

Reply:
Thanks for already applying the improvement.